client/src/components/User/UserBeaconSettings.vue

```diff
@@ -0,0 +1,259 @@
+      ><b v-b-modal.modal-beacon v-localize>Manage Beacon</b></a
+      >
+      <div v-localize class="form-text text-muted">Contribute variants to Beacon</div>
+      <b-modal
+          size="xl"
+          id="modal-beacon"
+          ref="modal"
+          ok-only
+          title="Manage Beacon"
+          title-tag="h1"
+          @show="loadSettings">
+
+        <!-- Explanation text-->
+        <p>
+          This setting will enable the possibility to share your Variants (in VCF (vcf) or compressed VCF (vcf_gz))
+          with the entire community in a secure way without sharing the entire file or Galaxy history. This is possible
+          by utilizing the <a href="https://beacon-project.io">Global Alliance for Genomics & Health Beacon Project</a>.
+          <br>
+          <br>
+          We turn Galaxy into a Beacon server and offer your variants anonymously via the Beacon Network. All shared
+          Variants will be merged to a single dataset. If someone searches in the Beacon Network for a specific variant
+          that is in one of your datasets our Galaxy server will return
+          <span class="cursive">“Yes, we have seen such a variant”</span>. Nothing more.
+          <br>
+          <br>
+          The user has the possibility to contact the Admin of the Galaxy server and we will in turn contact
+          you and ask if you want to contact this user and negotiate a potential data access.
+          Sharing is essential in Galaxy, secure sharing of sensitive data is now possible using Beacon.
+        </p>
+
+
+        <b-alert v-if="enabled" show>
+          <div class="flex-row space-between">
+            <div class="no-shrink">
+              Beacon sharing is <span class="bold">enabled</span> for your profile
+            </div>
+            <div class="fill"></div>
+            <div class="no-shrink">
+              <b-button @click="optOut" variant="danger">Disable</b-button>
+            </div>
+          </div>
+        </b-alert>
+
+
+        <!-- Setting to show when beacon is disabled -->
+        <b-alert v-if="!enabled" show>
+          <div class="flex-row space-between">
+            <div class="no-shrink">
+              Beacon sharing is currently <span class="bold">disabled</span> - no data will be shared
+            </div>
+            <div class="fill"></div>
+            <div>
+              <b-button @click="optIn" variant="success">Enable</b-button>
+            </div>
+          </div>
+        </b-alert>
+
+        <div v-if="enabled">
+          <p>
+            If you this setting you can share data by copying VCF/VCF.gz files to a history called
+            <span class="cursive">{{ beaconHistoryName }}</span>. The Beacon database is rebuild every day, this means
+            if
+            you disable the option here, of if we remove the history, or data in the history, the variants will
+            disappear
+            from the Beacon in the next 24h.
```

Comment:
```suggestion
            If you enable this setting you can share data by copying VCF/VCF.gz files to a history called
            <span class="cursive">{{ beaconHistoryName }}</span>. The Beacon database is rebuilt every day, which means
            if
            you disable the option here, or if you remove the history or data in the history, the variants will
            disappear
            from the Beacon in the next 24h.
```

Reply:
The same is actually true for the general description at the top of the window. Potentially, different Galaxy servers could build Beacon datasets with different granularity so they would have to modify the text.